chapters/en/chapter6/pre-trained_models.mdx

```diff
@@ -0,0 +1,199 @@
+# Pre-trained models for text-to-speech
+
+Text-to-speech task (also called _speech synthesis_) comes with a range of challenges.
+
+First, just like in the previously discussed automatic speech recognition, the alignment between text and speech can be tricky.  
+The model has to learn to generate the correct duration and timing for each phoneme, word, or sentence which can be challenging, 
+especially for long and complex sentences.
+
+Next, there's the long-distance dependency problem: language has a temporal aspect, and understanding the meaning of a 
+sentence often requires considering the context of surrounding words. Ensuring that the TTS model captures and retains 
+contextual information over long sequences is crucial for generating coherent and natural-sounding speech.
+
+Finally, training TTS models typically requires pairs of text and corresponding speech recordings. On top of that, to ensure 
+the model can generate speech that sounds natural for various speakers and speaking styles, data should contain diverse and 
+representative speech samples from multiple speakers. Collecting such data is expensive, time-consuming and for some languages 
+is not feasible. You may think, why not just take a dataset designed for ASR (automatic speech recognition) and use it for 
+training a TTS model? Unfortunately, automated speech recognition (ASR) datasets are not the best option. The features that 
+make it beneficial for ASR, such as excessive background noise, are typically undesirable in TTS. It's great to be able to 
```

Comment:
This is nicely explained!

A section on TTS datasets would be great here to expand on this a bit further (probably we can just cover the main TTS datasets like LJSpeech, Multilingual LibriSpeech, and VCTK and VoxPoupli). WDYT @Vaibhavs10? Wondering if you have a section similar to this already in your thesis that would be quick to adapt!

Reply:
There is also Libri-TTS/ LibriTTS-R - which is Multi Lingual Librispeech made more suitable for TTS - https://huggingface.co/datasets/cdminix/libritts-r-aligned (It is not officially maintained dataset by the way but works just as great.

```
LibriTTS is a multi-speaker English corpus of approximately 585 hours of read English speech at 24kHz sampling rate, prepared by Heiga Zen with the assistance of Google Speech and Google Brain team members. The LibriTTS corpus is designed for TTS research. It is derived from the original materials (mp3 audio files from LibriVox and text files from Project Gutenberg) of the LibriSpeech corpus. The main differences from the LibriSpeech corpus are listed below:
The audio files are at 24kHz sampling rate.
The speech is split at sentence breaks.
Both original and normalized texts are included.
Contextual information (e.g., neighbouring sentences) can be extracted.
Utterances with significant background noise are excluded.
```